luqezman/faceit-blacklist
# Add mongoose support to api

## Status
**READY**

## Migrations
NO

## Description
Add mongoose support to api

## Type of change


## Steps to Test or Reproduce

Before you start, make sure you have:

1. yarn dependencies installed
2. nx installed globally
3. docker running

```sh
# Starting docker
docker-compose up -t

# Starting api
nx serve api
```

Make sure the last log message contains `Listening at http://localhost:3333/api` after running the api

## Checklist:
